docs/platforms/python/tracing/instrumentation/custom-instrumentation/index.mdx

```diff
@@ -69,15 +69,32 @@ def eat_pizza(pizza):
     with sentry_sdk.start_transaction(op="task", name="Eat Pizza"):
         while pizza.slices > 0:
             eat_slice(pizza.slices.pop())
-
 ```
 
-<Alert title="Static & Class Methods" level="warning">
+<Alert title="Static & Class Methods" level="info">
 
 When tracing a static or class method, you **must** add the `@sentry_sdk.trace` decorator **after** the `@staticmethod` or `@classmethod` decorator (i.e., **closer** to the function definition). Otherwise, your function will break!
 
 </Alert>
 
+### Manually
+
+```python
+import sentry_sdk
+
+def eat_slice(slice):
+    ...
+
+def eat_pizza(pizza):
+    with sentry_sdk.start_transaction(op="task", name="Eat Pizza"):
+        while pizza.slices > 0:
+            span = sentry_sdk.start_span(description="Eat Slice")
```

Comment:
Is it necessary to call `__enter__` and `__exit__` here? We set some things on the scope in those magic methods, is that important?